.expeditor/scripts/release_habitat/package_and_upload_binary.sh

```diff
@@ -82,37 +82,36 @@ else
 fi
 
 echo "Compressing 'hab' binary"
-pushd "$build_dir" >/dev/null
-case "$BUILD_PKG_TARGET" in
-*-linux | *-linux-kernel2)
-    pkg_artifact="$tmp_root/results/${archive_name}.tar.gz"
-    tarball="$build_dir/$(basename "${pkg_artifact%.gz}")"
-    hab pkg exec core/tar tar cf "$tarball" "$(basename "$pkg_dir")"
-    hab pkg exec core/gzip gzip -9 -c "$tarball" > "$pkg_artifact"
-    ;;
-*-darwin | *-windows)
-    pkg_artifact="$tmp_root/results/${archive_name}.zip"
-    hab pkg exec core/zip zip -9 -r "$pkg_artifact" "$(basename "$pkg_dir")"
-    ;;
-*)
-    exit_with "$target_hart has unknown TARGET=$BUILD_PKG_TARGET" 3
-    ;;
-esac
-
+(cd "$build_dir"
+  case "$BUILD_PKG_TARGET" in
+  *-linux | *-linux-kernel2)
+      pkg_artifact="$tmp_root/results/${archive_name}.tar.gz"
+      tarball="$build_dir/$(basename "${pkg_artifact}.gz")"
```

Comment:
I think this needed to be `basename "${pkg_artifact}" .gz` (note the space).

Or, perhaps more readably, `basename --suffix=".gz" "${pkg_artifact}"`

Reply:
good call

Reply:
[That syntax isn't POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/basename.html), but maybe it doesn't matter here

Reply:
Well, poo... I like long options, but if we wanna stay POSIX, then that's cool too.